internal/orchestrator/healthcheck_setup_test.go

```
@@ -0,0 +1,202 @@
+		wantFatal    bool
+		wantLogin    string
+	}{
+		{"verified", HealthcheckCheckResult{Err: nil, Reachable: true, LoginURL: "https://hc/L"}, true, false, "https://hc/L"},
+		{"ready but not reachable -> retry", HealthcheckCheckResult{Err: nil, Reachable: false, LoginURL: "https://hc/L"}, false, false, "https://hc/L"},
+		{"auth fatal", HealthcheckCheckResult{Err: health.ErrHCAuth}, false, true, ""},
+		{"unknown fatal", HealthcheckCheckResult{Err: health.ErrHCUnknown}, false, true, ""},
+		{"disabled fatal", HealthcheckCheckResult{Err: health.ErrHCDisabled}, false, true, ""},
+		{"not ready retry", HealthcheckCheckResult{Err: health.ErrHCNotReady}, false, false, ""},
+		{"network retry keeps login", HealthcheckCheckResult{Err: errors.New("dial"), LoginURL: "https://hc/L2"}, false, false, "https://hc/L2"},
+		{"hostile ansi login dropped", HealthcheckCheckResult{Err: nil, Reachable: true, LoginURL: "https://hc/\x1b[2Jx"}, true, false, ""},
+		{"c1 control login dropped", HealthcheckCheckResult{Err: nil, Reachable: true, LoginURL: "https://hc/x"}, true, false, ""},
```

golangci-lint (2.12.2)

[error] ST1018: string literal contains the Unicode control character U+009B, consider using the '\u009b' escape sequence instead

(staticcheck)